server/routes.go

```diff
@@ -1180,11 +1180,12 @@ func (s *Server) GenerateRoutes() http.Handler {
 		})
 
 		r.Handle(method, "/api/tags", s.ListHandler)
-		r.Handle(method, "/api/version", func(c *gin.Context) {
-			c.JSON(http.StatusOK, gin.H{"version": version.Version})
-		})
 	}
 
+	// Use r.Any to call regular http.Handler handlers
+	// as we transition to a standard http.ServeMux.
+	r.Any("/api/version", gin.WrapF(s.versionHandler))
```

Comment:
Instead, the gin Mux can be wrapped by a normal mux, and that wrapper can consume it all over time.